packages/diagrams/frontend/sirius-components-diagrams/src/sprotty/views/RectangleView.tsx

```diff
@@ -108,6 +109,32 @@ export class RectangleView extends RectangularNodeView {
   }
 }
 
+const debugFontSize = 6;
+
+const styleDebug = {
+  fontSize: debugFontSize + 'px',
+  fill: 'green',
+};
+
+const debugInfos = (node: Readonly<Node>, size: { width: any; height: any }) => {
+  const diagram: Diagram = node.root as Diagram;
+  if (diagram.debug) {
+    return (
+      <g>
+        <text x={0} y={debugFontSize} style={styleDebug}>
+          x:{node.bounds.x.toFixed(2)} y:{node.bounds.y.toFixed(2)}
+        </text>
+        <text x={size.width} y={size.height - debugFontSize} style={styleDebug}>
+          Width: {size.width.toFixed(2)}
+        </text>
+        <text x={0} y={size.height - debugFontSize} style={styleDebug}>
+          Height: {size.height.toFixed(2)}
+        </text>
+      </g>
+    );
+  } else return null;
+};
+
```

Comment:
I've updated this code to have a better user experience with additional information and a better positioning of the data